src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+const filters = thead.firstElementChild.children;
```

Comment:
Maybe it's better to name this variable 'headers';

Reply:
filters or maybe sortFilters sounds more functional to me, header logically doesn't sort